python/TestHarness/testers/RunApp.py

```diff
@@ -21,6 +21,8 @@ def validParams():
     params.addParam('walltime',           "The max time as pbs understands it")
     params.addParam('job_name',           "The test name as pbs understands it")
     params.addParam('no_copy',            "The tests file as pbs understands it")
+    params.addStringSubParam('pbs_stdout', 'PBS_STDOUT', "Save stdout to this location")
+    params.addStringSubParam('pbs_stderr', 'PBS_STDERR', "Save stderr to this location")
```

Comment:
Please remove these lines - they shouldn't be valid parameters for normal RunApp Testers. You can still add parameters explicitly below in your logic. You don't need to pre-declare them here.